src/Dashboard/Grid.php

```diff
@@ -1106,6 +1110,14 @@ public function getAllDasboardCards($force = false): array
     {
         global $CFG_GLPI;
 
+        $cache_manager = new CacheManager();
+        $GLPI_CACHE = $cache_manager->getCoreCacheInstance();
+
+        $cards = $GLPI_CACHE->get('getAllDasboardCards', null);
+        if ($cards && !$force) {
```

Comment:
1. `null` is the default value of the param.
2. It is better to use strict checks.
```suggestion
        $cards = $GLPI_CACHE->get('getAllDasboardCards');
        if ($cards !== null && !$force) {
```